ironfish/src/rpc/routes/wallet/burnAsset.test.ts

```diff
@@ -74,6 +74,14 @@ describe('Route wallet/burnAsset', () => {
       })
       jest.spyOn(wallet, 'burn').mockResolvedValueOnce(burnTransaction)
 
+      const accountAsset = await account.getAsset(assetId)
+
+      expect(accountAsset).toBeDefined()
+
+      if (!accountAsset) {
+        throw new Error('accountAsset is undefined')
+      }
```

Comment:
references below line to accountAsset would not respect  the `expect(accountAsset).toBeDefined()` line as an undefined check and lead to a bunch of unnecessary conditional logic. So, I added this check here. I'm happy to change it back